### DISCUSSION TITLE
Becoming a contributor to 86box

### DISCUSSION THREAD

**Discussion by Tomeister1** [86Box/86Box#4460] [Q&A] [answered]
Hi I have been following 86box GitHub and using 86box for 3 years and I would like to add some IBM Aptiva roms machines from the retro web's website. Last night I tried to submit a pull request and that didn't work because I have to be a contributor in order to do that. So may I please if possible be a contributor to 86box?

Thanks.

**Comment by jriwanek** [comment, marked as answer]
That makes no sense at all, also, opening a pull request does *not* require contributor status.
